test/common/ssl/test_data/crl.cfg

```diff
@@ -0,0 +1,18 @@
+[ default ]
```

Comment:
Could we merge this into `ca_cert.cfg`?

Reply:
I am honestly not an expert when it comes to OpenSSL config files - I'm happy to take a stab at it if you'd like, but it'll take me a bit to familiarize myself with things, I suspect.

Reply:
I think moving content of this file into `ca_cert.cfg` should work, but skip `dir` and adjust `database` and `crlnumber` to avoid messing up other sub-commands.

If it doesn't "just work", then feel free to leave it as-is.

Reply:
Alright, that appears to have worked - thanks! 👍